apps/sms/test/marionette/lib/messages.js

```diff
@@ -264,23 +168,8 @@ var ConversationAccessor = require('./views/conversation/accessors');
           }
         },
 
-        addRecipient: function(number) {
-          this.Composer.recipientsInput.sendKeys(number + Chars.ENTER);
-
-          // Since recipient.js re-renders recipients all the time (when new
-          // recipient is added or old is removed) and it can happen several
-          // times during single "add" or "remove" operation we should
-          // wait until Recipients View is in a final state. The problem here is
-          // that between "findElement" and "displayed" calls element can
-          // actually be removed from DOM and re-created again that will lead to
-          // "stale element" exception.
-          var toField = this.Composer.toField;
-          toField.scriptWith(observeElementStability);
-          client.helper.waitFor(function() {
-            return toField.scriptWith(function(el) {
-              return !!el.dataset.__stable;
-            });
-          });
+        addRecipient: function(recipient) {
```

Comment:
Well essentially they do exactly the same thing, no matter how they are done internally, but we can revise it later anyway.